uint16[] current_quad
uint16[] current_hams
string last_name
uint8 age
uint32 score
